types/node/test/globals.ts

```diff
@@ -10,16 +10,6 @@
 
 declare var RANDOM_GLOBAL_VARIABLE: true;
 
-// global aliases for compatibility
```

Comment:
Don't these types still exist, and should be tested until their potential removal? Or are they tested another way?

Reply:
Their existence is _de facto_ tested by some residual usage around some older DT packages. If this needs to be more explicit, then could add a couple of quick checks for equivalence with their namespaced counterparts.

Reply:
I'd believe it, but it does seem cheap to just not delete the test, right?